app/views/users/_profile_header.html.erb

```diff
@@ -7,25 +7,35 @@
   .user-profile-follow-button {
     border: 1px solid <%= user_colors(@user)[:text] %> !important;
   }
-
   <% end %>
 </style>
-<div class="user-profile-header" style="<%= user_colors_style(@user) %>" itemscope itemtype="http://schema.org/Person" itemprop="mainEntity">
-  <meta itemprop="url" content="https://dev.to/<%= @user.username %>">
+
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Person",
+    "mainEntityOfPage": "True",
+    "url": "https://dev.to/<%= @user.username %>",
```

Comment:
This should be `user_url` similar to `article_url` used elsewhere in the PR.

Reply:
Ah, okay, I'll change it to, `<%= user_url(@user) %>`, right now! Good catch. 👀